content/en/docs/configuration/venafi.md

```diff
@@ -169,3 +169,10 @@ You are now ready to issue certificates using the newly provisioned Venafi
 
 Read the [Issuing Certificates](../../usage/certificate/) document for
 more information on how to create Certificate resources.
+
+# Issuer specific annotations
+
+## Custom Fields
+Starting `v0.14` you can pass custom fields to Venafi (TPP version `v19.3` and higher) using the `venafi.cert-manager.io/custom-fields` annotation on Certificate resources.
+The value is a JSON encoded array of custom field objects having a `name` and `value` key.
+For example: `venafi.cert-manager.io/custom-fields: '[{"name": "field-name", "value": "vield value"}]'`
```

Comment:
I think it may be useful to include a 'snippet' that includes the rest of the object too, e.g:

```
apiVersion: cert-manager.io/v1alpha2
kind: Certificate
metadata:
  name: example-com-certificate
  annotations:
    venafi.cert-manager.io/custom-fields: '[{"name": "field-name", "value": "vield value"}]'
...
```

It _may_ also be worth including more than one custom field in this example too? Although it may be a bit confusing all on one line.. I think this is also a valid way to write it though:

```
apiVersion: cert-manager.io/v1alpha2
kind: Certificate
metadata:
  name: example-com-certificate
  annotations:
    venafi.cert-manager.io/custom-fields: |-
      [
        {"name": "field-name", "value": "vield value"},
        {"name": "field-name-2", "value": "vield value"}
      ]
...
```

This has also just raised/made me think.. do we handle the case where multiple custom field entries specify the same `name`? What is the behaviour in that case? 🤔

Reply:
We do pass it in this way to Venafi, so their platform defines the behavior on multiple fields with the same name